platform/windows/detect.py

```diff
@@ -185,10 +185,11 @@ def get_opts():
         # Targeted Windows version: 7 (and later), minimum supported version
         # XP support dropped after EOL due to missing API for IPv6 and other issues
         # Vista support dropped after EOL due to GH-10243
+        # 7/8/8.1 support dropped after EOL due to GH-76829
         (
             "target_win_version",
-            "Targeted Windows version, >= 0x0601 (Windows 7)",
-            "0x0601",
+            "Targeted Windows version, >= 0x0A00 (Windows 10)",
+            "0x0A00",
         ),
```

Comment:
We could remove the option entirely, considering it'll be unviable to require Windows 11 for years and that the option can't even target Windows 11 in the first place (its version number is `10.x.x` just like Windows 10).

Reply:
Indeed, seems like [0x0A00 is last value](https://learn.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=msvc-170), so it probably can be removed.

Reply:
Removed.